.github/workflows/backend-ci.yml

```diff
@@ -9,7 +9,7 @@ env:
   CARGO_TERM_COLOR: always
   RUSTFLAGS: -D warnings
   REGISTRY: ghcr.io
-  RUST_VERSION: 1.70.0
+  RUST_VERSION: 1.81.0
```

Comment:
would also be good to add equivalent `rust-toolchain.toml` file in root:
```toml
[toolchain]
channel = "1.81.0"
profile = "default" # include rustfmt, clippy
```

Also while we are at it - can use current latest stable lts version
use `1.84.0` (i think)

Reply:
Good idea!